app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ActiveRecord::Base
+  def self.find_or_create_from_auth(data)
```

Comment:
Assignment Branch Condition size for find_or_create_from_auth is too high. [16.03/15]